levechad/settings.py

```diff
@@ -46,7 +46,8 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'bootstrapform',
-    'mathfilters'
+    'mathfilters',
+    'django_extensions',
```

Comment:
is it used?

Reply:
Yes - see answer above in your previous comment about this